src/graph_imported_curves.js

```
@@ -0,0 +1,98 @@
+      name: name,
+      points: points,
+    });
+    for (const point of points) {
+      this.minX = Math.min(point.x, _that.minX);
+      this.maxX = Math.max(point.x, _that.maxX);
+      this.minY = Math.min(point.y, _that.minY);
+      this.maxY = Math.max(point.y, _that.maxY);
+    }
+    curvesChanged();
+  }
+
```

ESLint

[error] Missing semicolon.

(semi)